winui/Masked-TextBox/Formatting-Value.md

```diff
@@ -7,4 +7,135 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Value mask formats in WinUI SfMaskedTextBox
```

Comment:
@Vignesh-SF3580 Learn all about how to set the mask format for the value in the MaskedTextBox (SfMaskedTextBox) control with prompts and literals.

Reply:
Changed the description.